src/SlackNotification/Types.ts

```diff
@@ -0,0 +1,39 @@
+import { EventObjectDecoderFailed } from './Event/Types';
+import { MissingEnvironmentVar } from './Environment';
+import { HttpError } from 'ajaxian';
+export type HandlerFail =
+  | MissingEnvironmentVar
+  | EventObjectDecoderFailed
+  | SlackNotifierError
+
+export type SlackNotifierError = HttpError | MessageDecoderFailed | SlackNotifierRequestFailed;
+
+export type SuccessLambdaResult = SlackNotificationSuccess | string;
+
+export interface SlackNotifierRequestFailed {
+  kind: 'slack-notifier-request-failed';
+  message: string;
+}
+
+export const slackNotifierRequestFailed = (message: string): SlackNotifierRequestFailed => ({
+  kind: 'slack-notifier-request-failed',
+  message,
+});
+
+export interface SlackNotificationSuccess {
+  kind: 'slack-notifier-request-succeeded';
+  message: unknown;
+}
+
+export const slackNotifierRequestSucceded = (s: unknown): SlackNotificationSuccess => ({
+  kind: 'slack-notifier-request-succeeded',
+  message: s  
+});
```

Comment:
message: string

Reply:
s is the name of the variable provided to the function, and because of how ajaxios works, it is unknown. @kofno looked into changing it to a string (to match ajaxian) but then decided not to. I don't remember why.

Reply:
I decided to leave it unknown because it is. Technically, the payload could be binary data (file downloads) which _can_ be represented as a string but would be meaningless.